projects/plugins/jetpack/tests/e2e/lib/flows/jetpack-connect.js

```diff
@@ -64,16 +64,18 @@ export async function doClassicConnection( mockPlanData ) {
 	return await ( await MyPlanPage.init( page ) ).returnToWPAdmin();
 }
 
-export async function doInPlaceConnection() {
+export async function doInPlaceConnection( plan = 'plan' ) {
```

Comment:
I find this default value (`plan = 'plan'`) a bit confusing.

Reply:
yeah, it's probably something that overlooked. `'plan'` does not make sense